deploy/examples/operator.yaml

```diff
@@ -620,7 +622,6 @@ spec:
             # If this is not set to true, the operator will watch for cluster CRDs in all namespaces.
             - name: ROOK_CURRENT_NAMESPACE_ONLY
               value: "false"
-
```

Comment:
nit: add back the empty line

Reply:
done

Reply:
Now remove the extra line 623

Reply:
done

Reply:
Did you push this change? The whitespace still looks different here

Reply:
now pushed for real with  extra blank line removed